src/Networks/Blockcore.Networks.x42/x42Setup.cs

```diff
@@ -114,7 +114,7 @@ internal class x42Setup
             HashGenesisBlock = "0x8cfb6674e48320009b1228644fab3fe850d53d073689365c1ecad55d8f77b504",
             HashMerkleRoot = "0x1d825dbc2ff5f9078885824347d231e54d72733779b0a3563b457c19368018ae",
             DNS = new[] { "testnet1.x42seed.host" },
-            Nodes = new[] { "18.138.31.117" },
+            Nodes = new[] { "161.97.135.78", "161.97.86.48"},
```

Comment:
@dimsavva @DennisAMenace  
Please review which is the correct testnet nodes, and i'll udpate if needed

Reply:
reverted